examples/rocket_okapi_example/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "sea-orm-rocket-okapi-example"
```

Comment:
@tyt2y3 I added a new folder to exemplify the usage of rocket_okapi. Hope this is enough otherwise I could make some changes you guys consider to make it better.